src/components/menu-bar/menu-bar.jsx

```diff
@@ -345,82 +345,86 @@ class MenuBar extends React.Component {
+                                        <MenuSection>
+                                            {this.props.canSave ? (
+                                                <MenuItem onClick={this.handleClickSave}>
+                                                    {saveNowMessage}
+                                                </MenuItem>
+                                            ) : []}
+                                            {this.props.canCreateCopy ? (
+                                                <MenuItem onClick={this.handleClickSaveAsCopy}>
+                                                    {createCopyMessage}
+                                                </MenuItem>
+                                            ) : []}
+                                            {this.props.canRemix ? (
+                                                <MenuItem onClick={this.handleClickRemix}>
+                                                    {remixMessage}
+                                                </MenuItem>
+                                            ) : []}
+                                        </MenuSection>
+                                    )}
+                                    <MenuSection>
+                                        <SBFileUploader
+                                            canSave={this.props.canSave}
+                                            userOwnsProject={this.props.userOwnsProject}
+                                            onUpdateProjectTitle={this.props.onUpdateProjectTitle}
+                                        >
+                                            {(className, renderFileInput, loadProject) => (
+                                                <MenuItem
+                                                    className={className}
+                                                    onClick={loadProject}
+                                                >
+                                                    {/* eslint-disable max-len */}
+                                                    {this.props.intl.formatMessage(sharedMessages.loadFromComputerTitle)}
+                                                    {/* eslint-enable max-len */}
+                                                    {renderFileInput()}
+                                                </MenuItem>
+                                            )}
+                                        </SBFileUploader>
+                                        <SB3Downloader>{(className, downloadProjectCallback) => (
                                             <MenuItem
                                                 className={className}
-                                                onClick={loadProject}
+                                                onClick={this.handleSaveToComputer(downloadProjectCallback)}
```

Comment:
Just a note — I know you didn't name this. This method name is misleading since it looks like an error. I would expect `onClick="{this.handleSaveToComputer}`, i.e. set the `onClick` handler to that method. But instead this is a function that returns a handler, so the name should probably reflect that.